presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetHiveRecordCursor.java

```diff
@@ -950,6 +950,7 @@ public Block getBlock()
         private final BlockConverter elementConverter;
 
         private BlockBuilder builder;
+        private int startingPosition;
```

Comment:
Probably no, because we don't know the number of elements we need to read (and fill in null for the ones we didn't read -- even complicated by nulls in between non-null values) in `ParquetListConverter`, unlike the struct or map converters. Cathcing nulls at the element level is simpler.